app/models/comment.rb

```diff
@@ -191,6 +191,37 @@ def remove_notifications
     Notification.remove_all_without_delay(notifiable_id: id, notifiable_type: "Comment", action: "Reaction")
   end
 
+  def calculate_score
```

Comment:
These methods needed to be made public so the job could access them.

Reply:
That's why I referred to this as a trick because you're exposing code that shouldn't be public just because you can trigger it at runtime. See my other comment for where to move this functionalities